src/main/groovy/org/mockito/release/internal/gradle/util/PomCustomizer.groovy

```diff
@@ -82,13 +85,14 @@ class PomCustomizer {
                 d.appendNode('url', "https://github.com/${split[0]}")
             }
 
-            def contributors = root.appendNode('contributors')
-            rootProject.ext.pom_contributors.each {
-                def split = it.split(':')
-                assert split.length == 2
-                def c = contributors.appendNode('contributor')
-                c.appendNode('name', split[1])
-                c.appendNode('url', "https://github.com/${split[0]}")
+            def contributorsNode = root.appendNode('contributors')
+            def fileName = Contributors.getAllProjectContributorsFileName(project.getBuildDir())
+            AllProjectContributorsReader reader = AllProjectsContributorsProvider.allProjectContributorsReader()
```

Comment:
@szpak In pom.xml are 2 sections: developers and contributors. Developers come from static configuration (see line above: `def developers = root.appendNode('developers')`). Contributors are fetched from GitHub. I will add some configuration property to disable this feature and combine contributors with old approach list.
I need also remove developers from contributors to don't duplicate them.